aws.go

```diff
@@ -118,3 +119,21 @@ func removePathFromRoleARN(arn string, path string) string {
 
 	return r.ReplaceAllString(arn, "/")
 }
+
+// Get AWS account ID
+func getAccountId() (string, error) {
+	cfg, err := config.LoadDefaultConfig(context.TODO())
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	client := sts.NewFromConfig(cfg)
```

Comment:
```suggestion
	client, err := getAWSClient()
	if err != nil {
		logger.Fatal("Unable to load SDK config, %v", zap.Error(err))
	}
```

Hey @fmubaidien , there is an internal method to get AWS config ⬆️